2022q4/NFS-in-prisons.adoc

```diff
@@ -6,27 +6,19 @@ link:https://people.freebsd.org/~rmacklem/nfsd-vnet-prison-setup.txt[Simple Setu
 
 Contact: Rick Macklem <rmacklem@freebsd.org>
 
-Several users of FreeBSD identified a need to run the NFS server inside
-a vnet prison.
+Several users of FreeBSD identified a need to run the NFS server inside a vnet prison.
 This turned into a small project, where I now have a patch that does this.
-It is currently available at the above link for testing or on phabricator
-as link:https://reviews.freebsd.org/D37519[D37519].
+It is currently available at the above link for testing or on phabricator as link:https://reviews.freebsd.org/D37519[D37519].
 Without this patch, the NFS server cannot be run in a prison.
 
-Not included in the above patch is the ability to run the
-man:rpc.tlsservd[8] or man:nfsuserd[8] daemons within the vnet prison.
-I do now have patches that allow these daemons to run in the
-vnet prison along with man:mountd[8] and man:nfsd[8], but I would
-like to get the above patch into main before adding support
-for man:rpc.tlsservd[8] or man:nfsuserd[8].
+Not included in the above patch is the ability to run the man:rpc.tlsservd[8] or man:nfsuserd[8] daemons within the vnet prison.
+I do now have patches that allow these daemons to run in the vnet prison along with man:mountd[8] and man:nfsd[8], but I would like to get the above patch into main before adding support for man:rpc.tlsservd[8] or man:nfsuserd[8].
 
 At this time, the code needs reviewing and testing.
-Hopefully this can be completed in the next few
-weeks, so that the patch can be committed to `main`
-and possibly also MFC'd to stable/13.
+Hopefully this can be completed in the next few weeks, so that the patch can be committed to `main` and possibly also MFC'd to stable/13.
 
 ==== To do
 
 * Testing the above patch
 * Reviewing the above patch
-* Doing the same for the man:rpc.tlsservd[8] and man:nfsuserd[8] patches
+* Doing the same for the man:rpc.tlsservd[8] and man:nfsuserd[8] patches.
```

Comment:
I would either drop the full stop on line 24 or add one to all the elements of the list: what we have here is a just a list, not a list into a sentence. Indeed, each element of the list starts with a capital letter.

So, if we want to make a sentence from each element list, a period is needed for each of them. Otherwise, none of them should have a period.

Reply:
_To do_ (ignoring the heading markup) is perceptually the beginning of a sentence, then: 

* beginning each bullet point with uppercase is probably Americanisation
  * I might seek a reference for this
* the example at <https://docs.freebsd.org/en/books/fdp-primer/book/#asciidoctor-unordered-lists> has no full stop, I doubt that this indicates that a list should _not_ end with a full stop
* an ending without a full stop appears **intensely** weird to my (UK English) eye
* the chapter at <https://docs.freebsd.org/en/books/fdp-primer/book/#writing-style> offers no guidance for lists that are presented as bullet points
  * status reports aside, I'll seek an American-friendly writing style point of reference for possible inclusion in the primer
* the rendered example at <https://docs.asciidoctor.org/asciidoc/latest/lists/unordered/#ex-deep> **does** use what appears to be a full stop at the end of a list with uppercase at the beginning of each point
  * no full stops for interim points
  * this appears OK to my (UK English) eye. 

![image](https://user-images.githubusercontent.com/192271/210244753-1d634b3a-d785-4de7-af14-577b77d9b85f.png)

Reply:
Resolved by <https://github.com/grahamperrin/freebsd-quarterly/commit/9270718fb9ddc104cfc4209780471cc67d6fcb22>.

In retrospect: the ending without a full stop appeared weirder in AsciiDoc than it does when rendered. 

From a UK point of reference that should (also) suit American readers, highlights added by me: 

----

![image](https://user-images.githubusercontent.com/192271/210251721-c656bd79-0a97-4a3d-8d1e-5224ac12e050.png)

----

Source: [Bullet points – Style.ONS](https://style.ons.gov.uk/house-style/bullet-points/) (2015-02-10)

* Office for National Statistics
* the example at the foot of the page.